src/components/Footer.tsx

```diff
@@ -0,0 +1,73 @@
+import React from 'react';
+import cn from 'classnames';
+import { Todo } from '../types/Todo';
+import { Status } from '../types/Status';
+
+type Props = {
```

Comment:
use interface prefer. 
Types have unique features but the interface is best for scaling.